### PR TITLE
Remove non-Castanets codes from BrokerCastanets::GetWritableSharedMemoryRegion.

### PR DESCRIPTION
**Pull request by joonghunpark**

### CODE REVIEW

File: mojo/core/broker_castanets.cc [hunk 1]
```diff
@@ -278,61 +278,6 @@ base::WritableSharedMemoryRegion BrokerCastanets::GetWritableSharedMemoryRegion(
       base::subtle::PlatformSharedMemoryRegion::CreateWritable(num_bytes);
   base::WritableSharedMemoryRegion r = base::WritableSharedMemoryRegion::Deserialize(std::move(region));
   return r;
```
Comment (hyowon) [inline mojo/core/broker_castanets.cc:280]: We have to remove this above code, or only when tcp_connection_ is true, we will do this direct allocation.

File: mojo/core/broker_castanets.cc [hunk 2]
```diff
@@ -278,61 +278,6 @@ base::WritableSharedMemoryRegion BrokerCastanets::GetWritableSharedMemoryRegion(
...
-
```
Comment (hyowon) [inline mojo/core/broker_castanets.cc:281]: The codes below are needed for original behavior even though we are using no-sandbox...

**Review by hyowon — CHANGES_REQUESTED** [review]
Wouldn't it be better keeping the existing code in Chromium?

**Comment by joonghunpark** [comment]
According to hyowon's comment, abandon this PR for now.

**Pull request closed without merging**